README.md

```diff
@@ -73,7 +73,7 @@ watch-files | (Blank) | Files to flag for modifications (e.g. package.json)
 watch-files-comment | (see [action.yml](./action.yml)) | Comment to leave on PR when watched files have been changed.
 
 [^1]: In a case where a contributor has a private membership to the org, the `ignore-authors` may be used to skip checks - however, this workflow file configuration could effectively make membership public.
-[^2]: Default regular expression is based on [linking to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). To keep PRs related to issues within the same same repo, we use the `KEYWORD #ISSUE-NUMBER` syntax. However, one could also use a regular expression like `(fix\|resolv\|clos)(e)*(s|d)* (my-org)\/([a-z0-9\-_]*)#\d+`
+[^2]: Default regular expression is based on [linking to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). To keep PRs related to issues within the same same repo, we use the `KEYWORD #ISSUE-NUMBER` syntax. However, one could also use a regular expression like `(fix(es|ed)?|((resolve|close)(s|d)?)) (my-org)\/([a-z0-9\-_]*)#\d*[1-9]\d*`
```

Comment:
seems like this footnote and the one after it are not working correctly in the preview. Note the escaping in the original regex.
https://github.com/mtfoley/pr-compliance-action/blob/7e77ac91d12627072e794fbce718e4dc82036086/README.md

Reply:
Escaping shouldn't be needed here as we are not in a table. The pervious escaping seems to incorrect.

The formatting issue seems to have because by the issue above.

Reply:
I think you're right. Thanks!